pkg/transforms/node.go

```diff
@@ -51,6 +51,14 @@ func NodeResourceBuilder(n *v1.Node) *NodeResource {
 	node.Properties["_systemUUID"] = strings.TrimRight(n.Status.NodeInfo.SystemUUID, "\000")
 	node.Properties["role"] = roles
 
+	conditions := n.Status.Conditions
+	for _, condition := range conditions {
+		if condition.Type == v1.NodeReady {
```

Comment:
Do we want to capture only the nodeReady status? For a node to be healthy, can others be relevant too - like, KubeletHasSufficientMemory/NoDiskPressure/SufficientPID?

Reply:
I had the same thought, these can be useful, but decided to match the CLI.
Also, it could be too much data to show on the table.
I could change it if you think we should.

Reply:
Maybe we should ask the customer what they mean by node health - do they look at all or one? For now, we can start with this one and add others as needed.